### PR TITLE
updated the CI failure message for clarity

### PR DESCRIPTION
**Pull request by grahamlopez**

### CODE REVIEW

File: tests/test_automation/jenkins_rhea_cpu.sh [hunk 1]
```diff
@@ -109,5 +109,14 @@ cp $BUILD_DIR/$BUILD_TAG.o* ../
 
 cp $BUILD_DIR/$BUILD_TAG.o* ../
 
-# explicitly check for correct test output from all builds
+# check for correct test output from all builds
+
+if [ $(grep '100% tests passed, 0 tests failed out of [0-9]*' ../$BUILD_TAG.o* | wc -l) -eq 4 ]
```
Comment (ye-luo) [inline tests/test_automation/jenkins_rhea_cpu.sh]: Sorry, it needs to be -ne 4.

Comment (grahamlopez) [reply tests/test_automation/jenkins_rhea_cpu.sh]: sigh, you are right. submitted fix. how embarrassing

**Comment by markdewing** [comment]
It would be nice to check the return values from cmake and make and give some guidance as to which stage started the failure (not for this PR, future idea).   Now it requires some searching to figure out where the failure happened.

**Comment by grahamlopez** [comment]
@markdewing agreed, there is quite a bit more we could be doing.

**Comment by prckent** [comment]
Please could whoever merges this retrigger https://github.com/QMCPACK/qmcpack/pull/1136 ? Hopefully it will work now.

**Pull request merged**